handlers/handlers.go

```diff
@@ -32,14 +31,21 @@ func init() {
 
 // CreateFilterID controls the creating of a filter idea when a new user journey is
 // requested
-func CreateFilterID(w http.ResponseWriter, req *http.Request) {
-	// TODO: This is a stubbed filter id - replace with real filter id from api once
-	// code has been written
-	filterID := rand.Intn(100000000)
-	fid := strconv.Itoa(filterID)
-
-	log.Trace("created filter id", log.Data{"filter_id": fid})
-	http.Redirect(w, req, "/filters/"+fid+"/dimensions", 301)
+func CreateFilterID(fil *filter.Client) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+
+		fid, err := fil.CreateJob("ds83jks0-23euufr89-8i3") //use a real dataset filter id
```

Comment:
Leave TODO comment, instead a general comment in code base. It'll be easier to search for when cleaning up service.